src/lib/constants.js

```diff
@@ -0,0 +1,9 @@
+//Import Assets
+import Image from "assets/images/sushiMochi.jpg";
+
+const DATACARROUSEL = [
```

Comment:
DATACARROUSEL no es un valor fijo, es decir, un valor no debería manipularse, es un mockup de datos e incluso como es un array, pueden ser modificados sus elementos, por lo que sería mejor cambiarlo a `camelCase`. Además, el nombre es mejor que refiera a lo que contiene, no a donde va a ser usado, porque eso puede variar.

```suggestion
const tipsData = [
```

Reply:
Done!